easybuild/easyblocks/o/openfoam.py

```diff
@@ -282,15 +307,16 @@ def sanity_check_step(self):
                                                     "refineMesh", "vorticity"]]
         # check for the Pstream and scotchDecomp libraries, there must be a dummy one and an mpi one
         if 'extend' in self.name.lower():
-            libs = [os.path.join(libsdir, x, "libPstream.so") for x in ["dummy", "mpi"]] + \
-                   [os.path.join(libsdir, "libscotchDecomp.so")]
+            libs = [os.path.join(libsdir, "libscotchDecomp.so")]
+            if LooseVersion(self.version) < LooseVersion('3.2'):
+                libs.extend([os.path.join(libsdir, x, "libPstream.so") for x in ["dummy", "mpi"]])
```

Comment:
hold on, did this actually disappear, or are you ignored something that went wrong during the build?

references?